### PR TITLE
built in function iter()  

### PR DESCRIPTION
**Pull request by nikolaspoczekaj**
### Description

Hey, thanks for assigning me. As my first contribution I worked on issue #2147.

### Type of Change

- Added iter.md

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/python/concepts/built-in-functions/terms/iter/iter.md
Comment (caupolicandiaz) [inline content/python/concepts/built-in-functions/terms/iter/iter.md:8]:
```suggestion
  - 'Functions'
```

File: content/python/concepts/built-in-functions/terms/iter/iter.md
Comment (caupolicandiaz) [inline content/python/concepts/built-in-functions/terms/iter/iter.md:30]:
```suggestion
The code below demonstrates the use of the `iter()` function:
```

File: content/python/concepts/built-in-functions/terms/iter/iter.md
Comment (caupolicandiaz) [inline content/python/concepts/built-in-functions/terms/iter/iter.md:33]:
```suggestion
# Create a list
```

File: content/python/concepts/built-in-functions/terms/iter/iter.md
Comment (caupolicandiaz) [inline content/python/concepts/built-in-functions/terms/iter/iter.md:36]:
```suggestion
# Get an iterator object
```

File: content/python/concepts/built-in-functions/terms/iter/iter.md
Comment (caupolicandiaz) [inline content/python/concepts/built-in-functions/terms/iter/iter.md:39]:
```suggestion
# Iterate over the list using the iterator
```

File: content/python/concepts/built-in-functions/terms/iter/iter.md
Comment (caupolicandiaz) [inline content/python/concepts/built-in-functions/terms/iter/iter.md:45]:
```suggestion
In the code above a list is declared, and the `iter()` function is used to get an iterator object for the list. Finally, the `next()` function is used to step through the items in the list.
```

Comment (caupolicandiaz) [reply content/python/concepts/built-in-functions/terms/iter/iter.md:45]: Generally, we try to avoid the use of first and second person pronouns in docs (I, you, we)

File: content/python/concepts/built-in-functions/terms/iter/iter.md
Comment (caupolicandiaz) [inline content/python/concepts/built-in-functions/terms/iter/iter.md:49]:
```suggestion
In the following example, the code is iterating and printing the items in the `fruits` list as in the previous example. However, in this instance a `while` loop is used to continue calling `next(fruit_iter)` until a `StopIteration` exception is raised, indicating that there are no more items to iterate over.
```

File: content/python/concepts/built-in-functions/terms/iter/iter.md
Comment (caupolicandiaz) [inline content/python/concepts/built-in-functions/terms/iter/iter.md:19]:
```suggestion
```

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@nikolaspoczekaj I've completed my initial review. Please update the entry, and tag me when you've finished making changes. Thanks

**Comment by nikolaspoczekaj** [comment]
@caupolicandiaz 

Done! Thanks for the quick review:)

**Review by caupolicandiaz — APPROVED** [review]
@nikolaspoczekaj Looks good, this is ready for a second review.

File: content/python/concepts/built-in-functions/terms/iter/iter.md
Comment (SSwiniarski) [inline content/python/concepts/built-in-functions/terms/iter/iter.md:15]:
```suggestion
The **`iter()`** function is a built-in Python function used to create an `iterator` object.
```

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@nikolaspoczekaj Looks good, I only had one comment to address.

**Comment by nikolaspoczekaj** [comment]
@SSwiniarski 

Okay, done. Thanks for the review :)

**Review by SSwiniarski — APPROVED** [review]

**Comment by SSwiniarski** [comment]
@nikolaspoczekaj Congrats on the new entry! Live link here: https://www.codecademy.com/resources/docs/python/built-in-functions/iter

image

**Comment by nikolaspoczekaj** [comment]
Thanks @SSwiniarski !

I'm looking forward to contributing more.

**Comment by yangc95** [comment]
Hey @nikolaspoczekaj! Thanks again for this entry. Our team of maintainers noticed that there is an overlap of terms. It appears that the `iter()` functions existed in the Iterators concept and this entry needs to be moved.

Would you be able to create a new pull request for edits? You'll be removing this folder/file path and doing a bit more critical thinking work by incorporating what you have in this entry, with the existing [iter()](https://www.codecademy.com/resources/docs/python/iterators/iter) entry. Please let me know if you have any questions or concerns!

**Comment by nikolaspoczekaj** [comment]
Hi @yangc95 , I'll take care of it :)

**Comment by SSwiniarski** [comment]
@nikolaspoczekaj Yes, that's the idea.

**Pull request merged**
